modules/persistent-storage-csi-snapshots-operator.adoc

```diff
@@ -24,7 +24,7 @@ The VolumeSnapshotContent CRD is not namespaced and is for use by a cluster admi
 
 VolumeSnapshot::
 
-Similar to the PersistentVolumeClaim CRD, the VolumeSnapshot CRD defines a developer request for a snapshot. The CSI snapshot controller handles the binding of a VolumeSnapshot object with an appropriate VolumeSnapshotContent object. The binding is a one-to-one mapping.
+Similar to the PersistentVolumeClaim CRD, the VolumeSnapshot CRD defines a developer request for a snapshot. The CSI Snapshot Controller Operator runs the CSI snapshot controller, which handles the binding of a VolumeSnapshot object with an appropriate VolumeSnapshotContent object. The binding is a one-to-one mapping.
```

Comment:
PersistentVolumeClaim should not be a CRD.

Reply:
@qinpingli I didn't realize that PVC is not a CRD. Is this okay now that I removed `CRD` from PVC?